lib/tasks/db/migrate_pending_staff_reply.rake

```diff
@@ -0,0 +1,30 @@
+namespace :db do
+  task migrate_pending_staff_reply: :environment do
+    ActsAsTenant.without_tenant do
+      # Run this after migrating posts to the new SubmissionQuestion topics.
+      migrate_pending_staff_reply
+    end
+  end
+end
+
+def get_pending_staff_reply
+  ActiveRecord::Base.connection.exec_query(<<-SQL)
+    SELECT actable_id FROM course_discussion_topics
+    WHERE actable_type='Course::Assessment::Answer'
+      AND pending_staff_reply=true
+  SQL
+end
+
+# Set pending staff reply based on the old topics
+def migrate_pending_staff_reply
+  old_topic_ids = get_pending_staff_reply
+  old_topic_ids = old_topic_ids.map { |x| x['actable_id'] }
+
+  old_topic_ids.each do |old_topic_id|
```

Comment:
I think you can actually:
 1. grab the `actable_id` and `question_id` for the `old_topic_ids` query. 
 2. grab the ids of all the relevant `Course::Assessment::SubmissionQuestion`
 3. grab all the objects with the `actable_ids` in 2, and `actable_type = Course::Assessment::SubmissionQuestion` and update_column to `true`